lib/youtubearchiver/video.rb

```diff
@@ -64,7 +64,7 @@ def parse(video_hash)
     def download_video
       return if @live
 
-      filename = "#{YoutubeArchiver.temp_storage_location}/#{SecureRandom.uuid}.mp4"
```

Comment:
In the other changed line, you determine the extension, but here it's just `.mp4` Would it be better to determine in both places?

Reply:
Oops, definitely. Agree.